mlir/lib/ExecutionEngine/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 # Exclude these from libMLIR.so because the JIT infrastructure
 # is a big dependency which most don't need.
 
+add_subdirectory(ROCm)
```

Comment:
should this be guarded by `MLIR_ROCM_RUNNER_ENABLED`?

Reply:
No, `MLIR_ROCM_RUNNER_ENABLED` is used to guard the generation of `mlir-rocm-runner` now. Even if `mlir-rocm-runner` is not enabled, we still need to populate this compilation unit because it will be used indirectly by `mlir-miopen-lib` where it lower everything down to ELF. I think it will become clear when I roll out my next PR.